racingcar-java/src/test/java/CalculatorTest.java

```diff
@@ -0,0 +1,50 @@
+import static org.assertj.core.api.Assertions.*;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+class CalculatorTest {
+
+    Calculator c = new Calculator();
+
+    @ParameterizedTest
+    @DisplayName("calculate 테스트")
+    @CsvSource(value = {"2 + 3 * 4 / 2=10", "3 * 10 - 6 / 2=12"}, delimiter = '=')
+    void calculateTest(String input, int result) {
+        assertThat(c.calculate(input)).isEqualTo(result);
+    }
+
+    @ParameterizedTest
+    @DisplayName("null 혹은 빈 공백 에러 테스트 ")
+    @NullAndEmptySource
+    void nullOrEmptyErrorTest(String input) {
+        assertThatIllegalArgumentException().isThrownBy(() -> c.calculate(input))
+                .withMessageMatching("input 값이 잘못되었습니다");
```

Comment:
```suggestion
        assertThatIllegalArgumentException()
                .isThrownBy(() -> c.calculate(input))
                .withMessageMatching("input 값이 잘못되었습니다");
```
하나의 메소드라인이기때문에 한칸 띄워주세요 :)